src/schema/users.ts

```diff
@@ -727,6 +764,11 @@ export const typeDefs = /* GraphQL */ `
     Get integrations for the user
     """
     userIntegrations: UserIntegrationConnection @auth
+
+    """
+    Get companies for user
+    """
+    companies: [UserCompany] @auth
```

Comment:
Just noticed this allows querying email for other users over `companies` field? Also do we want to limit in customRelation to something like 10 or 50 so we don't return hundreds of rows if someone tries to be naughty